File: Lunchtime/Controller/SetupPageThirdViewController.m
//
//  SetupPageFourthViewController.m
//  Lunchtime
//
//  Created by Willow Belle on 2015-11-15.
//  Copyright © 2015 Cosmic Labs. All rights reserved.
//

#import "SetupPageThirdViewController.h"
#import "LunchtimeLocationManager.h"

@interface SetupPageThirdViewController ()

@property (nonatomic, weak) IBOutlet UIButton *shareLocationPrivilegeButton;

@end

@implementation SetupPageThirdViewController

- (IBAction)shareLocationPrivilegeButtonPressed:(UIButton *)sender {
    [[LunchtimeLocationManager defaultManager] setup];
}

@end

File: Lunchtime/Controller/SetupPageThirdViewController.swift
//
//  SetupPageThirdViewController.swift
//  Lunchtime
//
//  Created by Willow Bumby on 2015-11-18.
//  Copyright © 2015 Lighthouse Labs. All rights reserved.
//

import UIKit

class SetupPageThirdViewController: UIViewController {

    // MARK: - Outlets
    @IBOutlet weak var sharePrivilegeButton: UIButton?

    // MARK: - Actions
    @IBAction func sharePrivilegeButtonPressed(sender: UIButton) {
        LunchtimeLocationManager.defaultManager().setup()
    }
}
